docs/support/managing-ckeditor-logo.md

```diff
@@ -0,0 +1,73 @@
+---
+category: licensing
+order: 30
+toc: false
+meta-title: CKEditor 5 support documentation
+meta-description: Learn how to get help and support and how to provide feedback.
+---
+
+# Managing the "Powered by CKEditor" logo
+
+## Why the "Powered by CKEditor" logo?
+
+Starting from version v38.0.0 onwards, all **open source installations** of CKEditor 5 (free users) carry a small “Powered by CKEditor” watermark in the bottom right corner of the editing area. The label links directly to the [CKEditor website](https://ckeditor.com/) and while it may be a little frustrating to some. This new branding approach is designed to make sure the entire community knows who is powering and modernizing their rich text editor.
+
+{@img assets/img/powered-by-ckeditor.png Placement of the "Powered by CKEditor" logo within the editor}
+
+It is also another way for us to gather feedback from users and continue to improve this major open source project that we have been supporting for more than 20 years. CKEditor has 40+ full time engineers and R&D professionals focused on its continual evolution, with 30K+ developers now using it on a daily basis.
```

Comment:
Numbers not synced with the text of #14082. I intentionally only mentioned those of us who contribute to the OSS part. Not all engineers.

Also, the 30k+ developers is definitely not right. IDK where that number came from but should be much much higher. Best not to include it.

Reply:
It all comes from here: [https://docs.google.com/document/d/1tDqiKTLUG0xKY_V3NGkgg6hVfGP-dIm96m8MDzm5baU/edit](https://docs.google.com/document/d/1tDqiKTLUG0xKY_V3NGkgg6hVfGP-dIm96m8MDzm5baU/edit)